rootfs/etc/nginx/lua/balancer.lua

```diff
@@ -28,11 +29,14 @@ local function balance()
   local backend = backends:get(backend_name)
```

Comment:
you can extract this to `get_current_backend`

Reply:
this is done at https://github.com/Shopify/ingress/pull/27, once you rebase with master you will have that change